<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
	<class>MainWindow</class>
	<widget class="QMainWindow" name="MainWindow">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>1021</width>
				<height>725</height>
			</rect>
		</property>
		<property name="windowTitle">
			<string>MainWindow</string>
		</property>
		<widget class="QWidget" name="centralwidget">
			<widget class="QLabel" name="label">
				<property name="geometry">
					<rect>
						<x>300</x>
						<y>290</y>
						<width>91</width>
						<height>21</height>
					</rect>
				</property>
				<property name="font">
					<font>
						<family>맑은 고딕</family>
						<pointsize>14</pointsize>
						<weight>75</weight>
						<bold>true</bold>
					</font>
				</property>
				<property name="text">
					<string>옷 이름 :</string>
				</property>
			</widget>
			<widget class="QLineEdit" name="lineEdit_clothname">
				<property name="geometry">
					<rect>
						<x>400</x>
						<y>290</y>
						<width>113</width>
						<height>25</height>
					</rect>
				</property>
			</widget>
			<widget class="QPushButton" name="pushButton">
				<property name="geometry">
					<rect>
						<x>530</x>
						<y>290</y>
						<width>91</width>
						<height>21</height>
					</rect>
				</property>
				<property name="text">
					<string>PushButton</string>
				</property>
			</widget>
		</widget>
		<widget class="QMenuBar" name="menubar">
			<property name="geometry">
				<rect>
					<x>0</x>
					<y>0</y>
					<width>1021</width>
					<height>21</height>
				</rect>
			</property>
		</widget>
		<widget class="QStatusBar" name="statusbar"/>
	</widget>
	<resources/>
	<connections/>
</ui>
